src/transformers/models/altclip/configuration_altclip.py

```diff
@@ -175,8 +167,6 @@ class AltCLIPVisionConfig(PretrainedConfig):
             The non-linear activation function (function or string) in the encoder and pooler. If string, `"gelu"`,
             `"relu"`, `"selu"` and `"gelu_new"` ``"quick_gelu"` are supported. layer_norm_eps (`float`, *optional*,
             defaults to 1e-5): The epsilon used by the layer normalization layers.
-        dropout (`float`, *optional*, defaults to 0.0):
```

Comment:
`hidden_dropout_prob` and `attention_probs_dropout_prob` are used, but not this one.